cypress/e2e/pdf.cy.ts

```diff
@@ -0,0 +1,34 @@
+/// <reference types = "Cypress" />
+
+import { urls } from "../config/urls";
+import { sharedFilesPaths, freeTestDataPaths } from "../support/paths";
+import { sharedFilesTestData, freeTestData } from "../support/testData";
+import { homePageSteps as homePageStepsShared } from "../pageobjects/sharedFiles/HomePageSteps";
```

Comment:
If you have shared steps I think you should name it homePageStepsShared and do not bother yourself with using "as" keyword